### PR TITLE
feat/add vita token

### PR DESCRIPTION
**Pull request by carlomigueldy**
- **feat: add vite dao token in enum**
- **feat: add vita token const**

### CODE REVIEW

**Comment by valentynkvasov** [comment]
at the current phase u need to add VITA to page.ts 
with token limits
image

**Comment by carlomigueldy** [comment]
> at the current phase u need to add VITA to page.ts with token limits image

Alright will add it

File: src/app/create/page.tsx [hunk 1]
```diff
@@ -79,6 +79,11 @@ const tokenLimits: TokenLimits = {
     max: 5000,
     step: 25,
   },
+  VITA: {
+    min: 39000,
```
Comment (valentynkvasov) [inline src/app/create/page.tsx:83]: oh let's decrease it to min: 150, max 3000, step: 25

since the price is 2+$ and min 39k that's HUGE))

Comment (carlomigueldy) [reply src/app/create/page.tsx:83]: Aight

**Pull request merged**